messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
Could you embed the call to the pexpect `which` in a function that will gracefully handle failure modes (e.g. matlab not on the path)